great_expectations/data_context/data_context/cloud_data_context.py

```diff
@@ -1,15 +1,46 @@
+import logging
+from typing import Mapping, Optional, Union
+
 from great_expectations.data_context.data_context.abstract_data_context import (
     AbstractDataContext,
 )
+from great_expectations.data_context.types.base import DataContextConfig, GeCloudConfig
 from great_expectations.data_context.types.data_context_variables import (
     CloudDataContextVariables,
 )
 
+logger = logging.getLogger(__name__)
+
 
 class CloudDataContext(AbstractDataContext):
     """
     Subclass of AbstractDataContext that contains functionality necessary to hydrate state from cloud
     """
 
+    def __init__(
+        self,
+        project_config: Union[DataContextConfig, Mapping],
+        runtime_environment: Optional[dict] = None,
+        ge_cloud_mode: bool = False,
```

Comment:
I thought about this too, but I'm concerned that this would be a breaking change for Cloud, which may expect to pass the value into the constructor.

This was why I was thinking of `self._ge_cloud_mode = True` that you saw below

 Given that we agree that we should get rid of this if we can, I'd like to check with Nimbus to see how they are instantiating DataContexts

Reply:
Please disregard previous comment.